local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -49,27 +50,20 @@ func (d *Deleter) DeletePVs() {
 			glog.Infof("Deleting PV %q", name)
 
 			// Cleanup volume
-			err := d.cleanupPV(pv)
+			err := d.deletePV(pv)
 			if err != nil {
 				cleaningLocalPVErr := fmt.Errorf("Error cleaning PV %q: %v", name, err.Error())
 				d.RuntimeConfig.Recorder.Eventf(pv, v1.EventTypeWarning, common.EventVolumeFailedDelete, cleaningLocalPVErr.Error())
```

Comment:
Event type should probably be an error

Reply:
OK

Reply:
@msau42 I looked into this more. It looks like core/v1 only defines constants for EventTypeNormal and EventTypeWarning, which is why the previous author of provisioner used warning in this situation and I copied it here. Let me know if there is a different constant that I could use here, I could not find another looking through k8s source code. The use cases seemed to be to use warning to report errors.

Reply:
Oh ok that's fine then